vec_inf/cli/_cli.py

```diff
@@ -18,163 +18,136 @@ def cli():
+    launch_cmd = f"bash {launch_script_path}"
 
-    models_df = load_models_df()
+    models_df = utils.load_models_df()
 
-    if model_name in models_df['model_name'].values:
-        default_args = load_default_args(models_df, model_name)
+    if model_name in models_df["model_name"].values:
+        default_args = utils.load_default_args(models_df, model_name)
         for arg in default_args:
             if arg in locals() and locals()[arg] is not None:
                 default_args[arg] = locals()[arg]
             renamed_arg = arg.replace("_", "-")
-            launch_cmd += f" --{renamed_arg} {default_args[arg]}" 
+            launch_cmd += f" --{renamed_arg} {default_args[arg]}"
     else:
         model_args = models_df.columns.tolist()
-        excluded_keys = ['model_name', 'pipeline_parallelism']
+        excluded_keys = ["model_name", "pipeline_parallelism"]
         for arg in model_args:
             if arg not in excluded_keys and locals()[arg] is not None:
                 renamed_arg = arg.replace("_", "-")
-                launch_cmd += f" --{renamed_arg} {locals()[arg]}"  
-    
-    output = run_bash_command(launch_cmd)
+                launch_cmd += f" --{renamed_arg} {locals()[arg]}"
+
+    output = utils.run_bash_command(launch_cmd)
 
     slurm_job_id = output.split(" ")[-1].strip().strip("\n")
     output_lines = output.split("\n")[:-2]
 
-    table = create_table(key_title="Job Config", value_title="Value")
+    table = utils.create_table(key_title="Job Config", value_title="Value")
     table.add_row("Slurm Job ID", slurm_job_id, style="blue")
     output_dict = {"slurm_job_id": slurm_job_id}
-    
+
     for line in output_lines:
         key, value = line.split(": ")
         table.add_row(key, value)
         output_dict[key.lower().replace(" ", "_")] = value
-    
+
     if json_mode:
         click.echo(output_dict)
     else:
         CONSOLE.print(table)
 
 
 @cli.command("status")
-@click.argument(
-    "slurm_job_id",
-    type=int,
-    nargs=1
-)
+@click.argument("slurm_job_id", type=int, nargs=1)
 @click.option(
     "--log-dir",
     type=str,
-    help='Path to slurm log directory. This is required if --log-dir was set in model launch'
+    help="Path to slurm log directory. This is required if --log-dir was set in model launch",
 )
 @click.option(
     "--json-mode",
     is_flag=True,
-    help='Output in JSON string',
+    help="Output in JSON string",
 )
-def status(slurm_job_id: int, log_dir: str=None, json_mode: bool=False) -> None:
+def status(
+    slurm_job_id: int, log_dir: Optional[str] = None, json_mode: bool = False
+) -> None:
     """
     Get the status of a running model on the cluster
     """
     status_cmd = f"scontrol show job {slurm_job_id} --oneliner"
-    output = run_bash_command(status_cmd)
+    output = utils.run_bash_command(status_cmd)
 
     slurm_job_name = "UNAVAILABLE"
-    status = "SHUTDOWN"
+    status: str = "SHUTDOWN"
```

Comment:
Any reason to add typing here?

Reply:
It was for debugging a `mypy` error. I've removed it.